client/src/Api.js

```diff
@@ -65,8 +65,12 @@ const Api = {
     },
   },
   questions: {
-    index() {
-      return instance.get('/api/questions');
+    index(questionnaireId) {
+      const options = {};
+      if (questionnaireId) {
+        options.params = { questionnaireId };
+      }
+      return instance.get('/api/questions', options);
```

Comment:
Here, we update the questions API index function so that it can take an optional questionnaireId and pass it to the API as a query parameter... the axios library instance will automatically take the params and hit the server with a url of the form `/api/questions?questionnaireId=[id number]`...